doc/extensions/authoring.md

```diff
@@ -133,3 +141,10 @@ See [Extension Metadata](metadata.md) for more information.
 
 
 Also, see the [FAQ](faq.md).
+
+### What's differencies between host and not host source code in Azure/azure-cli-extensions
+
+An advantage that hosting extension in [Azure/azure-cli-extensions](https://github.com/Azure/azure-cli-extensionsb) is that you could save the time to repeatedly build and upload the WHL file to pass some CI checks.
+
+As for hosting outside, you could easily provide a direct and explicit entry to end user with your customized introduction page, like [Azure DevOps Extension for Azure CLI](https://github.com/Azure/azure-devops-cli-extension), which is fit to popularize your Azure CLI extension if you plan to.
+Otherwise, user have to go deeper to find the detail page in Azure/azure-cli-extensions, such as `src/azure-firewall`, `src/vm-repair`.
```

Comment:
```suggestion
Otherwise, users have to go deeper to find the detailed page in Azure/azure-cli-extensions, such as `src/azure-firewall`, `src/vm-repair`.
```